src/modules/completed-task.js

```diff
@@ -0,0 +1,15 @@
+
+ function completed (checkbox, task, tasks) {
+  checkbox.addEventListener('change', () => {
+  if(checkbox.checked) {
+    task.completed = true;
+    console.log(task);
```

Comment:
kindly. My recommendation is not to use a console.log. Remember to use best practices for coding.